axes/helpers.py

```diff
@@ -342,7 +342,13 @@ def get_lockout_response(request, credentials: dict = None) -> HttpResponse:
         )
 
     if request.META.get("HTTP_X_REQUESTED_WITH") == "XMLHttpRequest":
-        return JsonResponse(context, status=status)
+            json_response = JsonResponse(context, status=status)
+            json_response["Access-Control-Allow-Origin"] = settings.AXES_ALLOWED_CORS_ORIGINS
+            json_response["Access-Control-Allow-Methods"] = "POST, OPTIONS"
+            json_response[
+                "Access-Control-Allow-Headers"
+            ] = "Origin, Content-Type, Accept, Authorization, x-requested-with"
```

Comment:
Why are these specific headers hardcoded for all users?

Reply:
@aleksihakli kind of the same question above, see previous answer.

Reply:
Roger.